libr/core/cmd_search.c

```diff
@@ -1039,9 +1041,11 @@ static RList *construct_rop_gadget(RCore *core, ut64 addr, ut8 *buf, int idx, co
 	}
 	int opsz = 0;
 	char *opst = NULL;
+int minopsz = 2;
 
 	while (nb_instr < max_instr) {
 		ht_uu_insert (localbadstart, idx, 1);
+		r_asm_set_bits (core->assembler, 16);
```

Comment:
uh, wat, is this always disassembling as 16 bit, that's not going to fly

Reply:
well. i think we had a config var to specify that we dont want cpu mode switches iirc